portfolio.css

```diff
@@ -0,0 +1,96 @@
+* {
+	margin: 0;
+	font-size: 20px;
+	color: #ff4716;
+	overflow: auto;
+}
+
+.navbar a {
+	padding: 50% 0 0 5%;
+	text-decoration: none;
+	margin: 50% 0 50% 0;
+	position: relative;
+	top: 30px;
+	font-size: 1.5rem;	
+}
+
+.navbar {
+	background-color: #feeeeb;
+	padding: 0 0 30px;
+}
+
+.welcome-banner {
+	background-image: url("https://images.pexels.com/photos/2132171/pexels-photo-2132171.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
+	width: 100%;
+	height: 100vh;
+	text-align: center;
+	background-size: cover;
+	background-repeat: no-repeat;
+}
+
+@media screen and (max-width: 800px) {
+	.welcome-banner {
+		height: auto;	
+}
+}
+
+h1 {
+	font-size: 2.5rem;
+	margin: 0 auto;
+	padding: 50px 0 200px 0;
+	color: #ffffff;
+}
+
+h2 {
+	text-align: center;
+	font-size: 1.5rem;
+}
+
+.about-alt {
+	background-color: #feeeeb;
+}
+
+.about-me-container {
+	text-align:center;
+	
```

Comment:
extra space here... I told you I was going to be a dick ^_^

Reply:
Fixed